composer.json

```diff
@@ -43,7 +43,8 @@
         "burgov/key-value-form-bundle": "to edit assoc multivalue properties. require version 1.0.*"
     },
     "conflict": {
-        "phpcr/phpcr-shell": "<1.0.0-beta1"
+        "phpcr/phpcr-shell": "<1.0.0-beta1",
+        "symfony/framework-bundle": "<2.3.27|>=2.6.0,<2.6.6"
```

Comment:
i don't think @lsmith77 intended to do that. the previous expression was `~2.3.27 || ^2.6.6 || ~3.0"` which would mean 2.3.*,>=2.3.27 or 2.6.6 or anything higher 2 or 3.*